builtins.go

```diff
@@ -664,6 +665,63 @@ func builtinMd5(i *interpreter, trace traceElement, x value) (value, error) {
 	return makeValueString(hex.EncodeToString(hash[:])), nil
 }
 
+func builtinBase64(i *interpreter, trace traceElement, input value) (value, error) {
+	var byteArr []byte
+
+	var sanityCheck = func(v int) (string, bool) {
+		if !(v < 256 && v >= 0) {
```

Comment:
A very minor nit: I think a more idiomatic way of checking if value is in the range would be:
`0 <= v && v < 256`. It's the closest you can get in most programming languages to the mathy notation 0 <= v < 256. 

Alternatively eliminating the negation would also make it more clear: `v < 0 || v > 256` than the current expresssion.

Reply:
I like the `||` variant, might be written as `0 > v || v > 256`

Reply:
no.. that is a little confusing to look at. I'm going to use `v < 0 || v > 255`